### PR TITLE
Create a countdown to 1st day of the year

### PR DESCRIPTION
**Pull request by RDjarbeng**
Added a countdown to 1st January or New year

### CODE REVIEW

**Review by nyakotey — COMMENTED** [review]
not bad, not bad at all :)

1. lots of commented section in <code>clock.js</code>. If you don't mind taking some out
2. the <code>startTime()</code> in the <code>clock.js</code> seems not be in use. if you can revisit that and take that out
3. if its possible, move the other time related functions in <code>app.js</code> that aren't coupled with any DOM element into <code>clock.js</code>

**Comment by RDjarbeng** [comment]
Made changes to the code in terms of readability

**Comment by RDjarbeng** [comment]
Made changes to the code in terms of readability

File: index.html [hunk 1]
```diff
@@ -1,52 +1,57 @@
...
             <div class="mode-row">
-                <span class="toggleMode"><i class="fas fa-sun"></i></span>
+                <span class="toggleMode" id='themeToggle'><i class="fas fa-sun"></i></span>
             </div>
             <div class="banner-row">
-                <!-- <div class="banner"> day <span class="day">X</span> / 365</div> -->
-                <div class="banner">CountDown to New Year <i class="fas fa-snowman"></i></div>
+
+                <div class="banner">
+                    <div class=""> Day <span class="day" id ='countDay'>X</span> / 365</div>
```
Comment (nyakotey) [inline index.html:24]: :(

**Comment by RDjarbeng** [comment]
You can improve the day count design if you want to 🙂

**Comment by RDjarbeng** [comment]
Fixed #30 
Added a quick authors page

**Review by nyakotey — COMMENTED** [review]
nice addition :)

**Pull request merged**